demo_usage/convert_tracking_dataset_to_forecasting.py

```diff
@@ -0,0 +1,196 @@
+
+import copy
+import csv
+import pdb
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple
+
+
+import numpy as np
+from argoverse.data_loading.simple_track_dataloader import SimpleArgoverseTrackingDataLoader
+from argoverse.data_loading.object_label_record import json_label_dict_to_obj_record
+from argoverse.utils.se3 import SE3
+from argoverse.utils.transform import quat2rotmat, quat_argo2scipy
+from recordclass import RecordClass
+from scipy.spatial.distance import cdist
+from scipy.spatial.transform import Rotation
+
+
+class Box3d(RecordClass):
+	center: np.ndarray # x, y, z
```

Comment:
If `center`s are always `np.ndarrays` of 3 elements, should we consider enforcing this with a type of `Tuple[float, float, float]`?